e2e/e2etests/test_erc20_withdraw.go

```diff
@@ -12,14 +12,11 @@ import (
 func TestERC20Withdraw(r *runner.E2ERunner, args []string) {
 	require.Len(r, args, 1)
 
-	approvedAmount := big.NewInt(1e18)
-
 	withdrawalAmount, ok := new(big.Int).SetString(args[0], 10)
 	require.True(r, ok, "Invalid withdrawal amount specified for TestERC20Withdraw.")
-	require.Equal(r, -1, withdrawalAmount.Cmp(approvedAmount))
 
-	// approve
-	tx, err := r.ETHZRC20.Approve(r.ZEVMAuth, r.ERC20ZRC20Addr, approvedAmount)
+	// approve 1 unit of the gas token to cover the gas fee
+	tx, err := r.ETHZRC20.Approve(r.ZEVMAuth, r.ERC20ZRC20Addr, big.NewInt(1e18))
```

Comment:
nit: we could create an `e2e/const.go` file which defines units, such as `const one = big.NewInt(1e18)`, so we reuse it everywhere we need it.

Reply:
Yeah we might have many values now for this. Created an issue: https://github.com/zeta-chain/node/issues/2540